content/contributing.md

```diff
@@ -0,0 +1,95 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.8
+  kernelspec:
+    display_name: Python 3
+    name: python3
+---
+
+<!-- #region id="QqmEP19zFVew" -->
+# Contributors Guide (DRAFT)
+<!-- #endregion -->
+
+<!-- #region id="lUOvQg1MGRIo" -->
+## A Brief Roadmap for First-time Contributors
+
+If you want to contribute to nx-guides and already decided on an algorithm to work on, here is a simple roadmap you can follow:
+
+1.   Clone `nx-guides` repository to your local machine.
+2.   Create a folder with the name of your algorithm under `nx-guides/content/algorithms`.
+3.   Create your markdown notebook under the folder you created in Step 2.
+4.  Create images and data folders under the folder you created in Step 2. If you use any static images and data, please put them under corresponding folders. (Optional)
+5.   Add your notebook's path to index.md file under `nx-guides/content/algorithms`.
+6.  When you complete your work and feel ready, push your changes to the repository and open a PR for review.
+<!-- #endregion -->
+
+<!-- #region id="4dKrGBiRIhYZ" -->
+## Some Tips
+<!-- #endregion -->
+
+<!-- #region id="vt0xsWsIDA1K" -->
+### 1. Your notebook should be a `.md` file.
+
+Your notebook should be in markdown notebook format. 
+
+If you normally use `.ipynb` notebooks to work on, you can convert them to `.md` using the following `jupytext` command:
+
+```
+jupytext --to markdown <notebook-name>.ipynb
```

Comment:
It's possible to use jupytext to do the conversion to myst specific markdown with:
```
jupytext  --to md:myst <notebook-name>.ipynb
```